roles/edpm_neutron_metadata/templates/wrappers/haproxy.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Looks like this file is not just similar but identical to what is used in the other dhcp role. I wonder if we could use a symlink here.

Reply:
It's now done in more "unified" way. I hope it's fine for You